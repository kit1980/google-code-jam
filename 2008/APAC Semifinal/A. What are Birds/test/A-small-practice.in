10
5
1000 1000 BIRD
2000 1000 BIRD
2000 2000 BIRD
1000 2000 BIRD
1500 2010 NOT BIRD
3
1500 1500
900 900
1400 2020
3
500 700 NOT BIRD
501 700 BIRD
502 700 NOT BIRD
2
501 600
502 501
1
100 100 NOT BIRD
3
107 93
86 70
110 115
10
1897 1802 BIRD
3765 3992 BIRD
1797 1802 NOT BIRD
1897 1702 NOT BIRD
3865 3992 NOT BIRD
3765 4092 NOT BIRD
1797 3992 NOT BIRD
1897 4092 NOT BIRD
3865 1802 NOT BIRD
3765 1702 NOT BIRD
8
1697 1802
1897 1602
3965 3992
3765 4192
1697 3992
1897 4192
3965 1802
3765 1602
10
885387 1000000 NOT BIRD
641422 516650 NOT BIRD
368691 1 NOT BIRD
1000000 897764 NOT BIRD
89173 383427 NOT BIRD
595369 5212 NOT BIRD
1 956430 NOT BIRD
665124 722863 NOT BIRD
513930 703136 NOT BIRD
723059 634023 NOT BIRD
10
898168 133070
1000000 961394
478043 175012
377374 176230
544920 484422
898538 1000000
1 575199
664371 798316
803527 566414
268981 776092
10
661314 621730 NOT BIRD
353896 616125 NOT BIRD
898815 100546 NOT BIRD
990365 515435 NOT BIRD
171088 1 NOT BIRD
1 117277 NOT BIRD
705404 393585 NOT BIRD
612400 392755 NOT BIRD
549677 95061 NOT BIRD
1000000 1000000 NOT BIRD
10
1 636227
297540 1
480571 140796
960379 651435
66602 97468
612903 710098
570493 573318
260757 1000000
560281 997302
1000000 724287
10
704568 698928 BIRD
872354 1000000 NOT BIRD
455307 1 NOT BIRD
28625 406220 NOT BIRD
805733 332872 NOT BIRD
530020 409504 NOT BIRD
1000000 363369 NOT BIRD
518150 226341 NOT BIRD
142619 700724 NOT BIRD
1 122847 NOT BIRD
10
243556 892922
597489 192380
888229 537765
792351 469842
441501 165194
87765 1
324915 470125
1 1000000
1000000 373744
148366 322228
10
908236 661238 NOT BIRD
894429 65819 NOT BIRD
1 231012 NOT BIRD
322405 1000000 NOT BIRD
114614 255764 NOT BIRD
436841 118607 NOT BIRD
235129 344819 NOT BIRD
67918 797370 NOT BIRD
1000000 466997 NOT BIRD
512184 1 NOT BIRD
10
295500 1
206726 689773
755591 1000000
268140 617506
469787 502955
338083 907670
1 308543
939508 110198
1000000 759356
278612 376349
10
44879 473731 NOT BIRD
178737 229321 NOT BIRD
1000000 248627 NOT BIRD
473417 586709 NOT BIRD
412925 213259 NOT BIRD
305625 442063 NOT BIRD
1 1 NOT BIRD
145551 1000000 NOT BIRD
900974 290072 NOT BIRD
684931 903882 NOT BIRD
10
845895 808934
370164 158661
387982 657200
252997 548900
713774 152960
1 272814
1000000 187191
116467 952927
911341 1000000
327685 1
10
402337 262010 BIRD
1000000 166343 NOT BIRD
697714 319302 NOT BIRD
401256 575322 BIRD
517722 44600 NOT BIRD
939812 955940 NOT BIRD
311706 1000000 NOT BIRD
829151 811128 NOT BIRD
763921 996659 NOT BIRD
1 1 NOT BIRD
10
54082 821397
740085 645631
1 679293
73851 207673
905386 1000000
1000000 741223
606043 306641
340714 783899
256191 352299
942591 280525
