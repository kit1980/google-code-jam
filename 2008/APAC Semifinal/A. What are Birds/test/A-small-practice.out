Case #1:
BIRD
UNKNOWN
NOT BIRD
Case #2:
UNKNOWN
NOT BIRD
Case #3:
UNKNOWN
UNKNOWN
UNKNOWN
Case #4:
NOT BIRD
NOT BIRD
NOT BIRD
NOT BIRD
NOT BIRD
NOT BIRD
NOT BIRD
NOT BIRD
Case #5:
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
Case #6:
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
Case #7:
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
NOT BIRD
NOT BIRD
UNKNOWN
NOT BIRD
UNKNOWN
NOT BIRD
Case #8:
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
Case #9:
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
UNKNOWN
Case #10:
UNKNOWN
NOT BIRD
UNKNOWN
UNKNOWN
NOT BIRD
NOT BIRD
UNKNOWN
UNKNOWN
UNKNOWN
NOT BIRD
